polygon-id/issuer/infrastructure/local/local/.vault/config/policies.hcl

```diff
@@ -0,0 +1,18 @@
+path "iden3/*" {
```

Comment:
this file looks out of place. We already have a local folder

Reply:
this is added in the new branch for amoy changes